src/utils/saml.ts

```diff
@@ -0,0 +1,27 @@
+import { isLeft } from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import { DOMParser } from "xmldom";
+
+const SAMLResponse = t.type({
+  RelayState: t.string,
```

Comment:
you can avoid declaring `RelayState` since you don't need it

Reply:
Fixed in dcd6147